index.js

```diff
@@ -542,6 +542,7 @@ ZipFile.prototype.openReadStream = function(entry, options, callback) {
       if (decompress) {
         var destroyed = false;
```

Comment:
This doesn't seem like it can be correct now because `destroyed` is never true, which causes errors to be duplicated.